warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/handler/shard/ShardedDataTypeHandler.java

```diff
@@ -655,9 +655,10 @@ protected Value createBloomFilter(RawRecordContainer event, Multimap<String,Norm
      * @param shardId
      * @param tableName
      * @param indexValue
+     * @param isReverse
      */
     protected Multimap<BulkIngestKey,Value> createTermIndexColumn(RawRecordContainer event, String column, String fieldValue, byte[] visibility,
-                    byte[] maskedVisibility, MaskedFieldHelper maskedFieldHelper, byte[] shardId, Text tableName, Value indexValue) {
+                    byte[] maskedVisibility, MaskedFieldHelper maskedFieldHelper, byte[] shardId, Text tableName, Value indexValue, boolean isReverse) {
```

Comment:
Might not be worth the effort here but we do have a `Direction` enum. Unfortunately it's in datawave-ws-client. We could bump that guy to datawave-core and then use it here. Using booleans as flags for things that don't really have "truthy" value to them sorta requires you to look at the method definition every time to remind yourself what true vs false means. 

```
createTermIndexColumn(..., indexValue, false); // requires drilling into the method to make sure you used the right value
```
VS
```
createTermIndexColumn(..., indexValue, Direction.FORWARD); // is unambiguously creating the forward values
```

Reply:
I'll work it into the follow-on commits, that sounds good.